python/servo/package_commands.py

```diff
@@ -193,24 +197,25 @@ def package(self, release=False, dev=False, android=None, debug=False, debugger=
                 raise Exception("Error occurred when getting Servo version: " + stderr)
             version = "Nightly version: " + version
 
-            template_path = os.path.join(dir_to_resources, 'Credits.rtf.mako')
-            credits_path = os.path.join(dir_to_resources, 'Credits.rtf')
+            template_path = path.join(dir_to_resources, 'Credits.rtf.mako')
+            credits_path = path.join(dir_to_resources, 'Credits.rtf')
             with open(template_path) as template_file:
                 template = mako.template.Template(template_file.read())
                 with open(credits_path, "w") as credits_file:
                     credits_file.write(template.render(version=version))
             delete(template_path)
 
             print("Writing run-servo")
-            bhtml_path = path.join('${0%/*}/../Resources', browserhtml_path.split('/')[-1], 'out', 'index.html')
-            runservo = os.open(dir_to_app + '/Contents/MacOS/run-servo', os.O_WRONLY | os.O_CREAT, int("0755", 8))
+            bhtml_path = path.join('${0%/*}/../Resources', browserhtml_path.split(os.sep)[-1], 'out', 'index.html')
```

Comment:
Also, see if you can get rid of the explicit `/`s in the first `path.join` in favor of multiple args.